lib/idp/constants.rb

```diff
@@ -30,6 +30,6 @@ module Constants
 
     MOCK_IDV_APPLICANT_WITH_SSN = MOCK_IDV_APPLICANT.merge(ssn: '900-66-1234').freeze
 
-    MOCK_IDV_APPLICANT_WITH_PHONE = MOCK_IDV_APPLICANT_WITH_SSN.merge(phone: '14065551234').freeze
+    MOCK_IDV_APPLICANT_WITH_PHONE = MOCK_IDV_APPLICANT_WITH_SSN.merge(phone: '12025551212').freeze
```

Comment:
The reason for changing this is that the default `user_with_2fa` uses this phone number, and it's assumed for the phone verification helper to use the user's own confirmed phone number to bypass the OTP check.

https://github.com/18F/identity-idp/blob/935cabee4a658767159e2b423889263de29b9da5/spec/support/request_helper.rb#L4-L6

https://github.com/18F/identity-idp/blob/935cabee4a658767159e2b423889263de29b9da5/spec/support/features/idv_step_helper.rb#L24-L27